src/_includes/newsletter-signup.njk

```diff
@@ -1,6 +1,6 @@
 {% callout %}
-<div class="newsletter-signup">
-	<h2 class="elv-h2 newsletter-signup-hed">Subscribe to the <a href="https://buttondown.email/11ty" class="elv-externalexempt">11ty Newsletter</a></h2>
+{#<div class="newsletter-signup">
```

Comment:
From personal experience I learned that it can help to add a comment above on why a piece of template is commented out (future you will thank you :-))

Reply:
Good call! We don’t need that widget any more, pushed another change to get rid of it (in favor of external newsletter signup link)